pallets/dao/src/lib.rs

```diff
@@ -287,8 +304,14 @@ pub mod pallet {
 		OrganizationAlreadyExists,
 		/// The user is not a member of this organization.
 		NotMember,
+		/// The user if over the maximum amount of organizations allowed to be affiliated with.
```

Comment:
We use the comments above the error message to display to the user. They should be descriptive of what the end user sees in the UI. 

For example:
/// You are not allowed to join more organizations at this time.

Reply:
https://github.com/UniversalDot/universal-dot-node/pull/96/commits/f81cfcd7f65c9ad8c93a8876011ca6fd2c0252b7